recipes/datamatrix/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "datamatrix" %}
+{% set pypiname = "python-" + name %}
+{% set version = "0.10.15" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
I think it would be nice to follow the same name as ``pypi``

Reply:
That would involve prefixing all packages with `python-` which comes from (and is valuable with) the Ubuntu package naming conventions. However, in conda this is quite superfluous as you have the build variable that indicates if something is a python package or not. In a previous package push of mine, a reviewer (can't remember who or where) actually advised against prefixing package names and recommended to keep them as elementary as possible, because you might get ambiguous package namings otherwise. For instance, with pyyaml you import the package with `import yaml`, but there is also a package named `yaml` available on conda. In the current case, if we add this package as `python-datamatrix`, someone may still add another new package called `datamatrix`, causing a similar confusion. I'm still ok with following your suggestion, but I would like to hear your thoughts on this.

Reply:
Any further suggestions on this?